bfdd/bfdd_cli.c

```diff
@@ -428,7 +428,8 @@ DEFPY_YANG(
 	}
 
 	if (!no && !bglobal.bg_use_dplane) {
-		vty_out(vty, "%% Current implementation of echo mode works only when the peer is also FRR.\n");
+		vty_out(vty,
```

Comment:
Two nits here:
1. The changed message should only be displayed in Linux:

   ```c
   #ifdef BFD_LINUX
   vty_out(vty, "%% Echo mode works correctly for IPv4, but only works when the peer is also FRR for IPv6.\n");
   #else
   vty_out(vty, "%% Current implementation of echo mode works only when the peer is also FRR.\n");
   #endif /* BFD_LINUX */
   ```

2. Capitalize `ipv4` to `IPv4` for consistency.